bin/run.js

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node
+"use strict";
+const { fork } = require('child_process');
+const path = require('path');
+console.log("CamScripter Launch");
+fork(path.normalize(__dirname + "/../dist/main.js"),{ detached: false, cwd: path.normalize(__dirname + "/../")});
```

Comment:
Fork je dulezity kvuli systemd?

Reply:
asi nechápu na co se přesně ptáš... tenhle soubor tu je aby se to dalo používat jako příkaz v CLI